test/units/testsuite-20.sh

```diff
@@ -151,8 +151,7 @@ test "$(systemctl show -P Result test20-true.scope)" = success
 runas() {
     declare userid=$1
     shift
-    # shellcheck disable=SC2016
-    su "$userid" -s /bin/sh -c 'XDG_RUNTIME_DIR=/run/user/$UID exec "$@"' -- sh "$@"
+    XDG_RUNTIME_DIR=/run/user/$(id -u "$userid") setpriv --reuid="$userid" --init-groups "$@"
```

Comment:
I guess `$(id ...)` should be wrapped with double quotations. Though I have no idea why shellcheck does not warn about that. Is the shellcheck now entirely disabled on our CI?

Reply:
Ok, thanks for the hints, all changes done.